### ISSUE TITLE
What is the maximum supported resolution of the model?

### ISSUE THREAD

**Issue by boolw** [ashawkey/RAD-NeRF#19]
ask a simple question first
Is the maximum supported resolution of the model 512*512? I want to try to run 1024*768 or even higher resolution on my RTX4090, where do I need to modify the code? Or does not require any code modification?

**Comment by ashawkey** [comment]
@boolw No, the NeRF works with arbitrary resolution and you don't need to modify anything. However, the required GPU memory will increase, and the training/inference speed will be slower since more pixels are being renderred.

**Comment by ashawkey** [comment]
Also, the off-the-shelf data preprocessing code (semantic segmentation, face tracking) may run into problems, but I'm not sure.

**Comment by boolw** [comment]
Great, thanks for your reply

**Comment by boolw** [comment]
I'm trying to drive the model with the text that chartGPT replies. But it has a long calling path:

text->TTS->wav->ASR->npy(logits/text)->RAD-NeRF

Such conversions seem inefficient and redundant.

Is there a better way to make it simpler and more efficient? ?

text->[???]->RAD-NeRF

For example, is it possible to use the Mel Spectrum output from TTS's am model to train or test the model? ?

**Comment by 91xiaoyang** [comment]
> 太好了，谢谢你的回复
Have you achieved it? His segmentation model doesn't seem to recognize people who are too far away

**Comment by RuojiWang** [comment]
> @boolw No, the NeRF works with arbitrary resolution and you don't need to modify anything. However, the required GPU memory will increase, and the training/inference speed will be slower since more pixels are being renderred.

how can i get 128x128 resolution output? cause the environmental memory i deploy can only support virtual digital people with 128x128 size. thanks for advance.
